docs/UserGuide.md

```diff
@@ -21,27 +21,30 @@ If you are looking to help develop this project, take a look at our [Developer G
 ## Table of Contents
 
 * [Introduction](#introduction)
-* [Layout]
-* [Using this guide]
+* [Layout](#layout)
+* [Using this guide](#using-this-guide)
+* [Command guide](#command-guide)
 * [Quick start](#quick-start)
-* [Features]
-  * Inventory management
+* [Tutorial](#tutorial)
+* [Features](#features)
+  * [Inventory management](#uinventory-managementu)
     * [Adding an inventory item: `addi`](#adding-an-inventory-item-addi)
-    * [Listing all inventory items: `list`](#listing-all-inventory-items-listi)
+    * [Listing all inventory items: `listi`](#listing-all-inventory-items-listi)
     * [Finding an inventory item: `findi`](#finding-an-inventory-item-findi)
     * [Deleting an inventory item: `deletei`](#deleting-an-inventory-item-deletei)
     * [Editing an inventory item: `editi`](#editing-an-inventory-item-editi)
-  * Order management
+  * [Order management](#uorder-managementu)
```

Comment:
Just wanted to clarify -- why does this link have the leading and trailing `u` characters?

Reply:
The trailing u characters were to underline those headers! just to distinguish them a bit from the actual commands